src/main/webapp/js/auscope/widgets/CustomRecordPanel.js

```diff
@@ -18,30 +18,9 @@ Ext.define('auscope.widgets.CustomRecordPanel', {
 
```

Comment:
there are a few blank lines and related formatting stuff in this file. Could you tidy it up a bit in this work (I know, it wasn't written by you but here is a chance to do some code cleanup)